DependencyInjection/Configuration.php

```diff
@@ -42,7 +42,12 @@ public function getConfigTreeBuilder()
                 ->arrayNode('access_denied_listener')
                     ->canBeEnabled()
                     ->beforeNormalization()
-                        ->ifArray()->then(function($v) { if (!empty($v) && empty($v['formats'])) { unset($v['enabled']); $v = array('enabled' => true, 'formats' => $v); } return $v; })
+                        ->ifArray()->then(function ($v) { if (!empty($v) && empty($v['formats'])) {
+     unset($v['enabled']);
```

Comment:
@lsmith77 I suggest you to revert these changes (and to open an issue in the tool)